### PR TITLE
[DE] Update `Client/Beatmap_editor/Menu`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
This article needed a rewrite because of grammar issues, some untranslated phrases and also the wrong pronouns (formal `Sie` instead of informal `du` which was chosen for the wiki)

- [x] Remove `needs_native_review` if PR gets reviewed
- [x] Check if `Beatmap-Thread` should be renamed to `Beatmap-Diskussion`
    - `Diskussion` is for the modding page, while `Thread` is for the post in the forum.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/Menu/de.md [hunk 1]
```diff
@@ -2,242 +2,242 @@
...
-| Stubs vrfwärts (`K`) | Bewegt Hit Objekt(e) um ein Beat nach vorne im Beat Snap Divisor. |
+| Stubs zurück (`J`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat zurück. |
+| Stubs vorwärts (`K`) | Bewegt Hit Objekt(e) abhängig vom Taktteiler um einen Beat nach vorne. |
 
 ## Ansicht
 
-![Ansichtmenü](img/M_View-DE.jpg "Ansichtmenü")
+![Ansichtsmenü](img/M_View-DE.jpg "Ansichtsmenü")
+
+Interessante Dinge.
```
Comment (The-Last-Cookie) [inline wiki/Client/Beatmap_editor/Menu/de.md:122]: This sounds too casual/incomprehensible when reading this again. In general, some phrases in the original version seem badly written, but this is not of concern here now. Despite this, I suggest to either remove the phrase entirely or change it to `Sehenswürdigkeiten`. If someone has a better idea how to translate just `places of interest`, please make a suggestion!

File: wiki/Client/Beatmap_editor/Menu/de.md [hunk 2]
```diff
@@ -2,242 +2,242 @@
...
+| Pausen zurücksetzen | Setzt alle Pausen zurück. Funktioniert nicht bei automatisch platzierten Pausen, die übermäßigen HP-Drain verhindern. |
 
-### Kommandos zum Stubsen
+### Befehle zum Stubsen
 
 | Name | Beschreibung |
 | :-- | :-- |
-| Stubs zurück (`J`) | Bewegt Hit Objekt(e) um ein Beat zurück im Beat Snap Divisor. |
-| Stubs vrfwärts (`K`) | Bewegt Hit Objekt(e) um ein Beat nach vorne im Beat Snap Divisor. |
+| Stubs zurück (`J`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat zurück. |
```
Comment (FamousPig) [inline wiki/Client/Beatmap_editor/Menu/de.md:115]:
```suggestion
| Stups zurück (`J`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat zurück. |
```

Checked the client to make sure.

File: wiki/Client/Beatmap_editor/Menu/de.md [hunk 3]
```diff
@@ -2,242 +2,242 @@
...
-| Alle Samples zurücksetzen | Entfernt alle festgelegten Hitsounds von den Hit Objekten. |
-| Kombo-Farben zurücksetzen | Setzt alle Kombofarben zurück. |
-| Pausen zurücksetzen | Setzt alle Pausen zurück. Funktioniert nicht bei automatisch platzierten Pausen. |
+| Samples des ausgewählten Objekts zurücksetzen | Entfernt den Hitsound vom ausgewählten Hit-Objekt. |
+| Alle Samples zurücksetzen | Entfernt alle festgelegten Hitsounds von den Hit-Objekten in dieser Schwierigkeitsstufe. |
+| Combo-Farben zurücksetzen | Setzt alle Combo-Farben zurück. |
+| Pausen zurücksetzen | Setzt alle Pausen zurück. Funktioniert nicht bei automatisch platzierten Pausen, die übermäßigen HP-Drain verhindern. |
 
-### Kommandos zum Stubsen
+### Befehle zum Stubsen
```
Comment (FamousPig) [inline wiki/Client/Beatmap_editor/Menu/de.md:111]:
```suggestion
### Befehle zum Stupsen
```

File: wiki/Client/Beatmap_editor/Menu/de.md [hunk 4]
```diff
@@ -2,242 +2,242 @@
...
 
 | Name | Beschreibung |
 | :-- | :-- |
-| Stubs zurück (`J`) | Bewegt Hit Objekt(e) um ein Beat zurück im Beat Snap Divisor. |
-| Stubs vrfwärts (`K`) | Bewegt Hit Objekt(e) um ein Beat nach vorne im Beat Snap Divisor. |
+| Stubs zurück (`J`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat zurück. |
+| Stubs vorwärts (`K`) | Bewegt Hit Objekt(e) abhängig vom Taktteiler um einen Beat nach vorne. |
```
Comment (FamousPig) [inline wiki/Client/Beatmap_editor/Menu/de.md:116]:
```suggestion
| Stups vorwärts (`K`) | Bewegt Hit Objekt(e) abhängig vom Taktteiler um einen Beat nach vorne. |
```

File: wiki/Client/Beatmap_editor/Menu/de.md (hunk 1 above)
Comment (FamousPig) [reply wiki/Client/Beatmap_editor/Menu/de.md:122]: The annoying part about removing lines is that it makes it harder to compare original and translation. Honestly I think it's best as is, but I would rather see it changed to `Sehenswürdigkeiten` than removed entirely.

File: wiki/Client/Beatmap_editor/Menu/de.md [hunk 5]
```diff
@@ -246,4 +246,4 @@ Diese Kommandos können in den [Einstellungen](/wiki/Client/Options) geaktiviert
 | Name | Beschreibung |
 | :-- | :-- |
 | Ingame-Hilfe anzeigen (`H`) | Öffnet im Editor ein Fenster mit hilfreichen Informationen. |
-| FAQ anzeigen | Sie werden nach [FAQ](/wiki/FAQ) geleitet. |
+| FAQ anzeigen | Du wirst zum [FAQ](/wiki/FAQ) weitergeleitet. |
```
Comment (FamousPig) [inline wiki/Client/Beatmap_editor/Menu/de.md:249]:
```suggestion
| FAQ anzeigen | Verlinkt nach [Beatmapping](/wiki/Beatmapping) |
```

https://github.com/ppy/osu-wiki/blob/fac8d562ef9bd43494421e3b57db5b305b044694/wiki/Client/Beatmap_editor/Menu/en.md?plain=1#L245

**Review by FamousPig — CHANGES_REQUESTED** [review]

File: wiki/Client/Beatmap_editor/Menu/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Client/Beatmap_editor/Menu/de.md:116]:
```suggestion
| Stups vorwärts (`K`) | Bewegt Hit-Objekt(e) abhängig vom Taktteiler um einen Beat nach vorne. |
```

**Review by FamousPig — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**